spec/util.lua

```diff
@@ -111,6 +111,7 @@ function util.tl_cmd(name, ...)
    assert(valid_commands[name], "not a valid command: tl " .. tostring(name))
 
    local cmd = {
+      string.format("LUA_PATH=%q", package.path),
```

Comment:
this is sufficient for CI because the environment is clean, but it might be safest to add all versioned variants (`LUA_PATH_5_2`, `LUA_PATH_5_3`, `LUA_PATH_5_4`) as well